_posts/2016-06-24-myeong-intro.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  "Myeong: introduction"
+date:   2016-06-24
+author: Myeong Lee
+---
+<link href="../assets/css/myeong.css" rel="stylesheet"></link>
+
+# Myeong Lee (Ph.D. Student)
+
+![photo](http://myeonglee.com/sites/default/files/myeong_2.jpg) I am a 2nd-year doctoral student at the University of Maryland, College Park studying information science (a.k.a., Maryland's iSchool). Specifically, I am interested in understanding how socio-technical factors affect various kinds of communities and organizations, and in designing social computing systems to solve community problems. This academic field is usually called Community/Urban Informatics, and it stretches to the fields of human-computer interaction (HCI) and computational social science depending on the focus of a study. I often use computational approaches to understanding the dynamics of local communities, organizations, and individuals' behavior based on various kinds of data (e.g., geo-tagged social media data), and want to further theorize how socio-technical factors shape the overall local information landscape in cities in the future. 

UW's DSSG this summer attracted me a lot, because the topics and skillsets that the program covered were perfectly fit to my interests and research directions. Especially, the project that I am working on, CrowdCensus, is an urban/social computing project that makes use of various kinds of geo-tagged data such as cellphone detail records (CDR) and social media data to predict socio-economic status of urban regions. Through this project, I am willing to build an easy-to-use model and system that can help policy-makers and urban planners, but also understand the data themselves so to better conceptualize geo-tagged data with a theoretical framework. I am really glad that I can work with brilliant people from diverse academic disciplines. 

I hope to learn how to deal with big urban data, how to connect conceptual variables to quantifiable measures, and how to build a robust and generalizable model for prediction. So far, I have learned how to speed up the computation using Python's pandas library, how to store and visualize geospatial data using PostgresSQL database and QGIS, and how to connect them each other. This series of lectures and trainings were very helpful for filling gaps in my knowledge about geospatial databases/visualizations.
```

Comment:
Maybe you could tell us a bit about why you're at the DSSG and what you hope to accomplish and/or learn over the course of the summer? If you want, you can tell us what you've been doing so far, and maybe what you've already learned?